api/SpeechSynthesisEvent.json

```diff
@@ -242,6 +242,58 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "milliseconds": {
+          "__compat": {
+            "description": "<code>elapsedTime</code> in milliseconds",
+            "support": {
+              "chrome": {
+                "version_added": "33"
+              },
+              "chrome_android": {
+                "version_added": "33"
+              },
+              "edge": {
+                "version_added": "14"
+              },
+              "firefox": {
+                "version_added": "49",
+                "version_removed": "95"
```

Comment:
Firefox for Linux is second even if old version. So you should add the note to explain that this is Windows and macOS implementation only.

Reply:
@hamishwillee In this case, I'd suggest two support statements:

- The first, with a version added but no version removed, tagged as partial, with a note saying Linux-only.
- The second, with a version added and a version removed, tagged as partial, with a note specifying Windows and macOS.